Makefile

```diff
@@ -706,9 +706,11 @@ release: clean-release ## Build and push container images using the latest git t
 	# Set the manifest image to the production bucket.
 	$(MAKE) manifest-modification REGISTRY=$(PROD_REGISTRY)
 	## Build the manifests
-	$(MAKE) release-manifests clean-release-git
```

Comment:
Previously we were cleaning up the patches before applying them, meaning the tagged image patch did not exist by the time `release-manifests-dev` is called.